source/PlayerInfoPanel.cpp

```diff
@@ -100,7 +101,8 @@ namespace {
 
 
 PlayerInfoPanel::PlayerInfoPanel(PlayerInfo &player)
-	: player(player), canEdit(player.GetPlanet())
+	: player(player), canEdit(player.GetPlanet()
+			&& !player.GetPlanet()->Attributes().count("requires: gaslining"))
```

Comment:
I prefer this to be handled in a more generic way; maybe something like `(player.GetPlanet().IsUnrestricted() || player.GetPlanet().HasSpaceport())` instead of the gaslining attributes check. Such a more generic solution also works if we get things like lava-blob-planets ("requires: volcanic/lava shields").

Reply:
See also Zitchas comment above...

Reply:
`IsUnrestricted` cannot be used here since the player cannot reorder ships only if he is "landed" on a planet on which you cannot really land (for example, a gas planet) and `IsUnrestricted` does not tell if that's the case.

Reply:
It would be better if there was a specific attribute which tells if you can actually land on a planet.

Reply:
Yes, such a special attribute would be even better.